pom.xml

```diff
@@ -309,6 +309,11 @@
         <version>${metrics.version}</version>
       </dependency>
       <dependency>
```

Comment:
keep alphabetical order (this should come after `javax.servlet` and before `log4j`)